OpenRA.Game/FluentProvider.cs

```diff
@@ -25,9 +27,25 @@ public static void Initialize(ModData modData, IReadOnlyFileSystem fileSystem)
 			lock (SyncObject)
 			{
 				modFluentBundle = new FluentBundle(Game.Settings.Player.Language, modData.Manifest.Translations, fileSystem);
-				mapFluentBundle = fileSystem is Map map && map.FluentMessageDefinitions != null
-					? new FluentBundle(Game.Settings.Player.Language, FieldLoader.GetValue<string[]>("value", map.FluentMessageDefinitions.Value), fileSystem)
-					: null;
+				if (fileSystem is Map map && map.FluentMessageDefinitions != null)
+				{
+					var files = Array.Empty<string>();
+					if (map.FluentMessageDefinitions.Value != null)
+						files = FieldLoader.GetValue<string[]>("value", map.FluentMessageDefinitions.Value);
+
+					string text = null;
+					if (map.FluentMessageDefinitions.Nodes.Length > 0)
+					{
+						var builder = new StringBuilder();
+						foreach (var node in map.FluentMessageDefinitions.Nodes)
+							if (node.Key == "base64")
+								builder.Append(Encoding.UTF8.GetString(Convert.FromBase64String(node.Value.Value)));
```

Comment:
Why are you encoding text as `base64`?

Reply:
The fluent grammar overlaps with MiniYaml, so cannot be directly included as yaml nodes without a horrible level of escaping. This feature is implemented for the RC, humans aren't meant to be manually constructing these (unless they *really* want to).

Reply:
https://github.com/OpenRA/OpenRA/pull/21622#issuecomment-2423927317

Reply:
It isn't streaming translations. It's including the fluent messages that are defined by the map, and are needed to correctly display the lobby options.

Otherwise, you end up with things like this:
<img width="225" alt="Screenshot 2024-10-20 at 13 56 47" src="https://github.com/user-attachments/assets/11ced1fb-9283-48c8-a4b7-ab14eb3a4da0">